Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -543,8 +543,9 @@ def fetch_incidents():
     now = int((datetime.utcnow() - datetime.utcfromtimestamp(0)).total_seconds() * 1000)
     last_run_object = demisto.getLastRun()
     last_run = last_run_object and last_run_object['time']
-    if not last_run:
-        last_run = now - 24 * 60 * 60 * 1000
+    if not last_run:  # first time fetch
+        last_run = now - parse_date_range(demisto.params().get('fetch_time', '3 days').strip(), to_timestamp=True)[0]
```

Comment:
1. Add UT, please
2. Did you manage to reproduce the bug locally?
3. What do you mean by `It happens because the incidents are not in the proper fetch time`? And can you explain why adding a first fetch time solves it?

Reply:
2. no.
3. the incident that was not fetched was a month old, the first_fetch is only 1 day ago. Hence, i've added the parameter.